{{cookiecutter.github_repository}}/provisioner/roles/nginx/defaults/main.yml

```diff
@@ -6,9 +6,24 @@ nginx_max_file_size: 10
 
 {% raw %}
 nginx_conf_file_name: '{{ project_name }}-{{ environment }}'
+
+# SSL common configurations
 ssl_cert_dir: '/etc/nginx/ssl'
 ssl_forward_secrecy_key_path: /etc/nginx/dhparam.pem
 ssl_forward_secrecy_key_length: 2048
 letsencrypt_ssl_cert_dir: '/etc/letsencrypt/live/{{ domain_name }}'
 letsencrypt_challange_root: '/app/certbot'
-letsencrypt_email: 'webmaster@{{ domain_name }}'{% endraw %}
+letsencrypt_email: 'webmaster@{{ domain_name }}'
+ssl_certificate: '{{ ssl_cert_dir }}/{{ domain_name }}/fullchain.pem'
+ssl_certificate_key: '{{ ssl_cert_dir }}/{{ domain_name }}/privkey.pem'
+ssl_dhparam: '{{ ssl_forward_secrecy_key_path }}'
+
+# Oldest compatible clients: Firefox 1, Chrome 1, IE 7, Opera 5, Safari 1, Windows XP IE8, Android 2.3, Java 7
+ssl_protocols: 'TLSv1 TLSv1.1 TLSv1.2'
+ssl_ciphers: 'ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA256:ECDHE-ECDSA-AES128-SHA:ECDHE-RSA-AES256-SHA384:ECDHE-RSA-AES128-SHA:ECDHE-ECDSA-AES256-SHA384:ECDHE-ECDSA-AES256-SHA:ECDHE-RSA-AES256-SHA:DHE-RSA-AES128-SHA256:DHE-RSA-AES128-SHA:DHE-RSA-AES256-SHA256:DHE-RSA-AES256-SHA:ECDHE-ECDSA-DES-CBC3-SHA:ECDHE-RSA-DES-CBC3-SHA:EDH-RSA-DES-CBC3-SHA:AES128-GCM-SHA256:AES256-GCM-SHA384:AES128-SHA256:AES256-SHA256:AES128-SHA:AES256-SHA:DES-CBC3-SHA:!DSS'
+ssl_session_timeout: '1d'
+ssl_session_cache: 'shared:SSL:50m'
+ssl_trusted_certificate: '{{ ssl_cert_dir }}/{{ domain_name }}/chain.pem'
+access_log_dir: '/var/log/nginx/{{ domain_name }}.access.log'
+error_log_dir: '/var/log/nginx/{{ domain_name }}.error.log'
```

Comment:
I think this should be called `error_log_file` and `access_log_file`.

Reply:
Done